examples/FactSet.AnalyticsAPI.Engines.Example/Examples/PAEngineMultipleUnitExample.cs

```diff
@@ -111,6 +111,11 @@ private static Configuration GetApiConfiguration()
                 Username = UserName,
                 Password = Password
             };
+            
+            // Uncomment below lines for adding the proxy configuration
+            //System.Net.WebProxy webProxy = new System.Net.WebProxy("http://myProxyUrl:80/");
+            //webProxy.Credentials = System.Net.CredentialCache.DefaultCredentials;
+            //_apiConfiguration.Proxy = webProxy;
```

Comment:
Can you please change the variable name to `_engineApiConfiguration`? Similarly at other places too.